docs/configuration.md

```diff
@@ -23,63 +23,116 @@ This section covers the client-side configuration.
 
 ### Stripe - Client-Side
 
-TODO: add documentation for client-side Stripe configuration
+To set up Stripe for client-side operations in OnLaunch:
+
+1. Sign up for a [Stripe](https://stripe.com/) account and get your publishable key.
+2. In your `.env` file, add your Stripe publishable key as `NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=<your_publishable_key_here>`.
+3. Stripe is disabled by default, so ensure `NEXT_PUBLIC_STRIPE_ENABLED` is set to `true` in your `.env` file to enable Stripe features.
 
 ## Server
 
 This section covers the server-side configuration.
 
 ### Database
 
-TODO: add documentation
+The `DATABASE_URL` environment variable configures the connection to the PostgreSQL database. It includes the username, password, host, port, and database name. Ensure this variable is set according to your production database credentials for deployment.
 
-### Emails
+For local development, the provided URL works with the default Docker Compose setup. For production, replace the username, password, host, and database name with your actual credentials.
 
-TODO: add documentation
+Example:
+```plaintext
+DATABASE_URL="postgresql://<username>:<password>@<host>:<port>/<database_name>?schema=public"
+```
 
 ### Free-Tier Subscription
 
-TODO: add documentation
-
-### Health
-
-TODO: add documentation
+If you have Stripe enabled, you can set the amount of maximum allowed requests for the free tier by `SUBSCRIPTION_FREE_VERSION_LIMIT=<request limit>`.
 
 ### NextAuth.js - Authentication
 
-TODO: add documentation
+This secret is used to encrypt the NextAuth.js JWT. For more information visit the [NextAuth.js Documentation](https://next-auth.js.org/).
 
 ### Redis
 
-TODO: add documentation
+Redis is used for caching and session storage, significantly improving the performance and scalability of the application. 
+
+```
+REDIS_ENABLED=true
+REDIS_HOST=<your_redis_host>
+REDIS_PASSWORD=<your_redis_password>
+REDIS_PORT=6379
+REDIS_SENTINEL_ENABLED=false
+REDIS_SENTINELS=<sentinels_if_any>
+REDIS_SENTINEL_NAME=<sentinel_name_if_any>
+REDIS_CACHE_MAX_AGE=60 // cache max age in minutes
+```
 
 ### Sentry - Server-Side
 
-TODO: add documentation
+Similar to Sentry on the client-side, you can see how the parameters are used in `sentry.server.config.tx`. 
 
 ### Signup
 
-TODO: add documentation
+The config value of `SIGNUPS_ENABLED` (default: `false`) is used to allow new signups to the system.
 
 ### SMTP
 
-TODO: add documentation
+OnLaunch uses SMTP to send emails. Configure the SMTP server details with the prefix `SMTP_` in your `.env` file:
+
+- `SMTP_HOST`: The host address of your SMTP server.
+- `SMTP_PORT`: The port your SMTP server uses.
+- `SMTP_USER` and `SMTP_PASS`: The username and password for SMTP authentication, if required.
+- `SMTP_FROM_EMAIL_ADDRESS` and `SMTP_FROM_NAME`: The email address and name that will appear as the sender.
+
+Ensure these values are set to match your email service provider's requirements.
+
+### Email Templates
```

Comment:
The headers where sorted alphabetically.
Did you move it on purpose?

Reply:
yes, in my opinion, alphabetically sorting makes only sense when the reader expects specific headlines. The reader will not know about a headline for "email templates" or "templates for emails". But i would expect the config doc to reflect the order of parameters in the .env file, or at least group different sections that belong together actually next to each other like the ones with the `SMTP_` prefix